libs-scala/resources-akka/src/main/scala/com/digitalasset/resources/akka/AkkaResourceOwnerFactories.scala

```diff
@@ -15,4 +15,7 @@ trait AkkaResourceOwnerFactories[Context] {
 
   def forMaterializer(acquire: () => Materializer): AbstractResourceOwner[Context, Materializer] =
     new ActorMaterializerResourceOwner(acquire)
+
+  def forMaterializer(system: ActorSystem): AbstractResourceOwner[Context, Materializer] =
+    new ActorMaterializerResourceOwner(() => Materializer(system))
```

Comment:
Is this used anywhere?

Reply:
Yes, in PR 3/4 of this series ([here](https://github.com/digital-asset/daml/pull/8607/files#diff-921a400b092b6dc2af4ec6cb976108d2fb39f8021596e187590a6709352c5a30)).

Reply:
I think I'd prefer if you added it there, then.

I think it might be nice for this to take a `() => ActorSystem` and have it call `createActorSystem().flatMap(actorSystem => forMaterializer(...))` so you can just write:

```
           materializer <- ResourceOwner.forMaterializer(createActorSystem).acquire()
```

We almost never use the `actorSystem`.

Reply:
I'll remove this from here and see how to address this in the follow-up PR.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/8604/commits/fab1ac3ef1ed92d81451909be1ae0fcc2570e52f